src/elements/content-sidebar/SidebarNav.js

```diff
@@ -59,6 +59,8 @@ const SidebarNav = ({
         onClick: onBoxSignClick,
         status: boxSignStatus,
         targetingApi: boxSignTargetingApi,
+        blockedReason: boxSignBlockedReason,
```

Comment:
Minor, but we generally try to alphabetize our props, attributes, etc. when possible. I also wonder if it's time to move the less critical property accessors, like `blockedReason `, `targetingApi`, etc., into the SidebarNavSign component itself.